sklearn/ensemble/forest.py

```diff
@@ -153,6 +156,12 @@ def __init__(self,
         self.verbose = verbose
         self.warm_start = warm_start
         self.class_weight = class_weight
+        self.allow_missing = missing_values is not None
```

Comment:
This doesn't work with grid search and pipeline because GridSearchCV find allowed parameters with get_params method rather than passing these as inputs. Indeed, GridSearchCV instantiates the class with all default parameters and after that, it changes the values. Then allow_missing will be always equals to false. I don't know the best solution to resolve this issue. Maybe add allow_missing value as a constructor argument ?